assets/js/components/notifications/dashboard-auth-alert.js

```diff
@@ -24,22 +24,23 @@ import { __, sprintf } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import { getModulesData } from '../../util';
 import Notification from '../notifications/notification';
 
 const DashboardAuthAlert = () => {
 	const { admin: { connectURL } } = global.googlesitekit;
-	const { currentAdminPage } = global.googlesitekit.admin;
-	const product = currentAdminPage
-		.replace( /googlesitekit|module|-/g, ' ' )
-		.replace( /(^\w{1})|(\s{1}\w{1})/g, ( match ) => match.toUpperCase() )
-		.trim();
+	const { requiredScopes, grantedScopes } = global.googlesitekit.setup;
+
+	const productUrl = requiredScopes.filter( ( e ) => ! grantedScopes.includes( e ) )[ 0 ];
+	const productSlug = productUrl ? productUrl.toString().split( '/' ).pop() : '';
+	const product = getModulesData()[ productSlug ] ? getModulesData()[ productSlug ].name : '';
 
 	return (
 		<Notification
 			id="authentication error"
-			title={ __( 'Site Kit can’t access necessary data', 'google-site-kit' ) }
+			title={ __( "Site Kit can't access necessary data", 'google-site-kit' ) }
 			/* translators: %1$s: Product name */
-			description={ sprintf( __( 'Site Kit can’t access the relevant data from %1$s because you haven’t granted all API scopes requested during setup. To use Site Kit, you’ll need to redo the setup for %1$s – make sure to approve all API scopes at the authentication stage.', 'google-site-kit' ), product ) }
+			description={ sprintf( __( "Site Kit can't access the relevant data from %1$s because you haven't granted all API scopes requested during setup. To use Site Kit, you'll need to redo the setup for %1$s – make sure to approve all API scopes at the authentication stage. ", 'google-site-kit' ), product ) }
```

Comment:
This was probably done automatically by your editor, but these lines should be restored to the previous versions here https://github.com/google/site-kit-wp/pull/1586/commits/43ee72e9959db04816a871222f749abc4f81c342

Reply:
Updated.